pkgs/applications/video/gpu-screen-recorder/gpu-screen-recorder-notification.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  pkg-config,
+  meson,
+  ninja,
+  libX11,
+  libXrender,
+  libXrandr,
+  libXext,
+  libglvnd,
+  gitUpdater,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gpu-screen-recorder-notification";
+  version = "1.0.0";
+
+  src = fetchurl {
+    url = "https://dec05eba.com/snapshot/${pname}.git.${version}.tar.gz";
+    hash = "sha256-cyM3y1MYZNEHKyiyAqYXcSwZe0tcnPNukxErm/dIo6Y=";
+  };
+
+  sourceRoot = ".";
+
+  postPatch = ''
+    substituteInPlace depends/mglpp/depends/mgl/src/gl.c \
```

Comment:
setting LD_LIBRARY_PATH with a wrapper should work

Reply:
But why have the overhead of a wrapper when you don't need it? :)